resources/splunk_service.rb

```diff
@@ -0,0 +1,158 @@
+# Cookbook Name:: cerner_splunk_ingredient
+# Resource:: splunk_service
+#
+# Resource for managing Splunk as a system service
+class SplunkService < ChefCompat::Resource
+  include CernerSplunk::PlatformHelpers, CernerSplunk::ResourceHelpers,
+          CernerSplunk::ServiceHelpers, CernerSplunk::RestartHelpers
+
+  resource_name :splunk_service
+
+  property :name, String, name_property: true, identity: true
+  property :package, [:splunk, :universal_forwarder], required: true
+  property :options, String, default: ''
```

Comment:
Both of those have to do with accepting the EULA and performing first run activities as opposed to launching the service. Should that be a different action for the service resource? Or does it fit with another resource? (Or at the first point we would execute a command, we can look for the ftr file in the installation directory and do it for them?)